<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt_libs demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_page_1">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>Page 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_page_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>36</height>
         </size>
        </property>
        <property name="text">
         <string>Page 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_1">
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>10</y>
            <width>204</width>
            <height>32</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_tip_label_test">
           <item>
            <widget class="QPushButton" name="btn_test_tip_label_1">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Test tip label 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_test_tip_label_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Test tip label 2</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QPushButton" name="play_button">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>120</y>
            <width>85</width>
            <height>44</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Play</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/images/music-player-play.png</normaloff>:/icons/images/music-player-play.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_3">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>120</y>
            <width>85</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="resource.qrc">
            <normaloff>:/icons/images/plus-black-symbol.png</normaloff>:/icons/images/plus-black-symbol.png</iconset>
          </property>
         </widget>
         <widget class="QListWidget" name="listWidget">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>110</y>
            <width>256</width>
            <height>192</height>
           </rect>
          </property>
          <item>
           <property name="text">
            <string>List widget item 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>List widget item 2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>List widget item 3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>List widget item 4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>List widget item 5</string>
           </property>
          </item>
         </widget>
         <widget class="QScrollBar" name="horizontalScrollBar">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>210</y>
            <width>271</width>
            <height>16</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>260</y>
            <width>271</width>
            <height>21</height>
           </rect>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QWidget" name="">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>60</y>
            <width>671</width>
            <height>29</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Combox item 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 5</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2">
             <property name="editable">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Combox item 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Combox item 5</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="text">
              <string>CheckBox</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,10">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="lineEdit_input"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>OK</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QWidget" name="widget_keyboard_container" native="true"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_vertical_stub">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_stub">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
